sdk/src/lib/ik-image/ik-image.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, AfterViewInit, OnInit, ElementRef, Input, OnChanges } from '@angular/core';
+import { ImagekitService } from '../imagekit-angular.service';
+import { Dict, QueryParameters, IkImageComponentOptions } from '../utility/ik-type-def-collection';
+import { getInt, namedNodeMapToObject, setElementAttributes } from '../utility/utils';
+import { buildSrc, getResponsiveImageAttributes, SrcOptions } from '@imagekit/javascript';
+import type { Transformation } from '@imagekit/javascript'
+
+@Component({
```

Comment:
Components are not marked as standalone, preventing direct import in standalone Angular applications (14+).

The SDK claims standalone support but only provides half the solution - it has the provider function but the components themselves aren't standalone. Consumers must still import the entire ImagekitAngularModule even in standalone apps, which defeats the purpose of Angular's modern standalone API.